posts/_markdown/quantumalgorithmsorg.md

```diff
@@ -0,0 +1,41 @@
+- Being the peer-reviewed, up-to-date, go-to reference for lemmas, theorems, corollaries, and algorithms needed by quantum algorithms researchers and quantum software developers.
+
+As of today, the book focuses on an algorithmic perspective of quantum machine learning and touches topics like quantum algorithms for Monte Carlo, techniques for lower bounding query complexity, and numerical experiments on real datasets. A sizeable appendix also covers the theory needed to understand quantum machine learning from a computer science perspective. This is how the content is structured so far.
+
+**In Part 1** - *introduction and toolkit* - we briefly introduce our formalism for quantum computing, with the axioms of quantum mechanics, explaining how to I/O (input and output) classical data in a quantum computer. Then, we outline a list of theorems and algorithms that are the basic toolkit of the quantum algorithm researcher and give concrete examples on how to use them. You can find in here all the subroutines for performing singular value estimation, phase estimation, finding the minimum, distance estimation, inner product estimation, amplitude estimation, and so on.
+
+**In Part 2** - *quantum algorithms and quantum machine learning* - we see how to use the techniques described previously, with a focus on machine learning. We see how to perform clustering (q-means, quantum Gaussian mixture models) and dimensionality reduction (quantum PCA and quantum slow feature analysis). Last but not least, we have a chapter on quantum algorithms for Monte Carlo (which we are expanding in these weeks to include more algorithms and applications), algorithms on graphs, and techniques for lower bounding the query complexity of your algorithms, and much more. 
+
+**In Part 3** - *the appendix* - we focus on putting together the mathematical background, definitions, and theorems that are used throughout the book. We put much emphasis on linear algebra, concentration inequalities, and tricks that are often taken for granted in most papers. 
+
+
+The book has a core team and many contributors that help keep it updated, clear, and correct. 
+The book started in its early days from the old blog of Alessandro Luongo, subsequently including parts of his Ph.D. thesis on quantum machine learning. Then we added contents from the MSc thesis of Armando Bellante. We recently hosted five students from the mentorship program of the Quantum Open Source Foundation. With them, we worked on quantum algorithms on graphs, quantum algorithms for speeding up Monte Carlo techniques, and quantum perceptrons algorithms. Many friends and great researchers have contributed to the project by spotting typos, writing chapters, submitting parts of their papers or thesis chapters. We wholeheartedly thank them and report their names here: Patrick Rebentrost, Yassine Hamoudi, Martin Plávala, Trong Duong, Filippo Miatto, Jinge Bao, Michele Vischi, Adrian Lee, Samantha Buck, Sathyawageeswar Subramanian. 
```

Comment:
Maybe vary the starts of the sentences on l23 and l24

Reply:
Changed in the new commit! :)